src/System.Text.Json/porting_guide/README.md

```diff
@@ -0,0 +1,173 @@
+# Guide to Porting from Newtonsoft.Json to System.Text.Json
+
+## Overview
+
+* The `System.Text.Json` APIs that are included in .NET Core 3.0 are designed primarily for performance critical scenarios and hence emphasize runtime performance and reducing allocations over high usability and convenience. Furthermore, the in-box JSON stack does not aim to provide feature parity with `Newtonsoft.Json`, especially since it is in its infancy. That said, we still want to make it easier for those whose needs can be fulfilled by the new APIs and those who are looking for better performance. This guide captures some of the common pitfalls and programming patterns that are useful to note when trying to use the new in-box APIs.
```

Comment:
please add some basic perf comparison numbers to very briefly highlight how much of a benefit a user will have when switching to from Newtonsoft.Json to System.Text.Json.